docs/tutorials/pydy_for_autolev_users.rst

```diff
@@ -0,0 +1,672 @@
+|                       | dynamicsymbols(‘u1’)  | differentiate between |
+|                       |                       | variables,            |
+|                       | u2 =                  | motionvariables and   |
+|                       | dynamicsymbols(‘u2’)  | specifieds during     |
+|                       |                       | declaration. Instead, |
+|                       |                       | it takes different    |
+|                       |                       | lists of these as     |
+|                       |                       | parameters in objects |
+|                       |                       | like the KanesMethod. |
++-----------------------+-----------------------+-----------------------+
+| Imaginary j           | j = symbols(‘j’)      | I is a sympy object   |
+|                       |                       | which stands for the  |
+|                       | j = I                 | imaginary unit. One   |
+|                       |                       | can define complex    |
+|                       |                       | numbers using it.     |
+|                       |                       |                       |
+|                       |                       | z = x + I*y           |
+|                       |                       |                       |
+|                       |                       | where x, y and z are  |
+|                       |                       | symbols.              |
++-----------------------+-----------------------+-----------------------+
+||                      ||                      ||                      |
+| Tina = 2*pi           | Tina = 2*pi           | Numerical constants   |
+|                       |                       | like pi can be        |
+|                       |                       | imported from mpmath. |
++-----------------------+-----------------------+-----------------------+
+||                      ||                      ||                      |
+| abs(x)^3 + sin(x)^2 + | abs(x)**3 + sin(x)**2 |                       |
+| acos(x)               | + acos(x)             |                       |
++-----------------------+-----------------------+-----------------------+
+| E = (x+2*y)^2 +       | E = (x+2*y)**2 +      | For more information  |
+| 3*(7+x)*(x+y)         | 3*(7+x)*(x+y)         | refer to              | 
+|                       |                       | `simplification. <htt |
+| Expand(E, n:m)        | expand(E)             | p://docs.sympy.org/la |
+|                       |                       | test/tutorial/simplif |
+| Factor(E, x)          | factor(E)             | ication.html>`__      |
+|                       |                       |                       |
+| Coef(y, x)            | y.coeff(x)            |                       |
+|                       |                       |                       |
+| Replace(y, sin(x)=3)  | y.subs([(sin(x), 3)]) |                       |
```

Comment:
SymPy has `xreplace` and `subs`. `xreplace` is exact replacement, any idea what Autolev does? exact or mathematical replacement?

Reply:
Also Autolev does things inplace here, right? And SymPy doesn't, so you'd need: `y = y.xreplace({sin(x): 3})`.

Reply:
I'm guessing it does exact replacement because Autolev has an Evaluate function which does mathematical replacement.  Autolev seems to return in some instances and do things inplace in some cases. It is inplace here though so I'll change that.